aws/modules/deploy/charts/config.yml

```diff
@@ -7,19 +7,32 @@
 # Child directory named "values" may contain ERB template with custom values
 # for the chart, that is being rendered in `rake generate` task.
 #
-# Values file may also contain the following structure to set name and
-# namespace values that will be passed to Helm:
+# Every chart should have the following attributes:
 #
-# chart-metadata:
-#  name: install_name
-#  namespace: install_namespace
+# beautiful-chart:
+#  release_name: release-name-to-pass-to-helm
+#  release_namespace: release-namespace-to-pass-to-helm
 #
 
 # TODO: switch couchdb deployment to Helm chart
-# - couchdb
-- nginx-ingress
-- cert-manager
-- gpii-dataloader
-- gpii-preferences
-- gpii-flowmanager
-# - fluentd
+# couchdb:
+#   release_name: couchdb
+#   release_namespace: gpii
+nginx-ingress:
+  release_name: nginx-ingress
+  release_namespace: gpii
+cert-manager:
+  release_name: cert-manager
+  release_namespace: kube-system
+gpii-dataloader:
+  release_name: dataloader
```

Comment:
I understand the introduction of these attrs and specifically of `release_namespace`. Cool.

Tell me more about `release_name`. If it exists solely so that in certain places we can use `dataloader` instead of `gpii-dataloader`, I think I'd rather live with the extra `gpii-` rather than introduce the complexity of this extra mapping. Does `release_name` serve some other purpose?

Reply:
I would prefer to keep `release_name` attribute functionality, since it gives us more flexibility: custom deployment names, multiple deployments of the same helm chart, etc. It is not mandatory, system will take chart name if `release_name` is not present. I feel like this is really low priority, since it affects only AWS part.

Reply:
ok